### DISCUSSION TITLE
Support for ActronAir ESP Platinum QUE

### DISCUSSION THREAD

**Discussion by ryanfurtner** [awulf/Actron485#1] [Q&A]
I was wondering how different the ultima system is to the ActronAir ESP Platinum QUE that I have? I have some form of Android wall controller for my system, but it is reliant on actron servers for remote control. I would love to ditch it completely and have it HA integrated, completely local.

**Comment by reubec2259** [comment]
Hi Mate

I’m new to the whole home ESP32 stuff. Could you give me a quick run down on how this integrates to the esp, were the wifi code sits how the ESP32 is seen by home assistant. Thanks man sorry for the hassle

**Comment by awulf** [comment]
Sorry about the delayed response. But I don't know if the QUE system will work with this Actron485 module. They also have no official MODBUS controller either unlike all the older and newer systems.
It might be worth just connecting to the A/B with J6 port on the indoor PCB, and seeing what comes out. There would at least be some data that won't be supported, as e.g. the individual wall zone controls temperature set point can be remotely controlled. I would lean towards that it's completely different and won't work at all, so packets would need to be reverse engineered.
